lib/js-api/src/api/content-rest-api/model/nodeChildAssociation.ts

```diff
@@ -49,6 +49,8 @@ export class NodeChildAssociation {
     permissions?: PermissionsInfo;
     definition?: Definition;
     association?: ChildAssociationInfo;
+    reason?: string;
```

Comment:
As per the other comment `getHolds` should return something else so these changes can be reverted

Reply:
so generally form /`file-plans/{filePlanId}/holds` endpoint we can get 
```
{
  id: string,
  name: string,
  reason: string,
  description: string
}
```
But actually we don't need `reason` and `description` for our purpose (if something will change we also can add those properties) and if users want they can use `legalHoldApi.getHolds` to get all data (full interface)

Reply:
I understand that but actually as far as I can see the `listParents` method returns `NodeAssociationPaging` which contains `NodeAssociation` so I'm not sure if this is the right class to be modified

Reply:
created `HoldPaging`